src/libmongoc/tests/unified/runner.c

```diff
@@ -359,6 +358,7 @@ test_file_new (test_runner_t *test_runner, bson_t *bson)
    bson_parser_array_optional (
       parser, "createEntities", &test_file->create_entities);
    bson_parser_array_optional (parser, "initialData", &test_file->initial_data);
+   bson_parser_doc_optional (parser, "_yamlAnchors", &test_file->yaml_anchors);
```

Comment:
Parsing this is not necessary as per the unified test format spec:
> This is only used to define anchors within the YAML files and MUST NOT be used by test runners.

Reply:
With the way that the `bson_parser` works, it is necessary to add this field to the `parser` here to avoid it complaining.  We still aren't using `_yamlAnchors` anywhere, though.